core/templates/pages/exploration-editor-page/translation-tab/translation-tab.component.spec.ts

```diff
@@ -235,6 +253,7 @@ describe('Translation tab component', () => {
 
   afterEach(() => {
     component.ngOnDestroy();
+    fixture.destroy();
```

Comment:
are we still sure that the error might occur again if the component is not destroyed?

Reply:
removed** while testing i have added this code.